### ISSUE TITLE
Modbus API Break?

### ISSUE THREAD

**Issue by martiniil** [PilzDE/pilz_robots#379]
### Commit
b5d88cf6b3d0628ec25a5720522ed4a7d213c4fe

### Steps to reproduce
```
roslaunch prbt_moveit_config moveit_planning_execution.launch sim:=false
rosservice call /prbt/execute_braketest
rosservice call /prbt/brake_test_required
```

### Expected behavior
Result is `-1`

### Observed behavior
Result is `1`, like before the braketest.

**Comment by martiniil** [comment]
In #312 the new api should be documented.

**Comment by ct2034** [comment]
The values that correlate to the operation modes have also changed. In our control cabinet (in relation to the labels on pitmode)

**Comment by martiniil** [comment]
I started implementing the changes: #385

**Comment by jschleicher** [comment]
Is this break on ROS side or on PSS4k side? `-1` looks more like a bug in the implementation (uninitialized) than an update to the specification needing a new version?

**Comment by jschleicher** [comment]
Implemented in #385
